pca/utils/dependency_injection.py

```diff
@@ -9,26 +10,26 @@
 
 
 class Container:
-    def __init__(self):
+    def __init__(self, default_scope: 'Scopes'=None):
```

Comment:
PEP8 for annotations (here and further):
`def __init__(self, default_scope: 'Scopes' = None):`